.github/workflows/ci.yml

```diff
@@ -10,38 +10,43 @@ jobs:
     name: Test
     runs-on: ubuntu-latest
 
-    strategy:
-      fail-fast: false
-
     steps:
       - uses: actions/checkout@master
+      - id: install
+        uses: actions-rs/toolchain@v1
         with:
-          fetch-depth: 2
-
-      - name: Install stable Rust
-        run: rustup update stable && rustup default stable
-
-      - name: Set rustc version
-        run: echo "::set-env name=CURRENT_RUSTC_VERSION::$(rustc -V)"
+          toolchain: stable
+          override: true
+      - uses: actions-rs/cargo@v1
+        with:
+          command: install
+          args: cargo-sweep
```

Comment:
`cargo-sweep` should be installed after restoring `~/.cargo/bin` so that we can use the cached build of it.